src/GitLink/Pdb/SrcSrvContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SrcSrvContext.cs" company="CatenaLogic">
+//   Copyright (c) 2014 - 2016 CatenaLogic. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace GitLink.Pdb
+{
+    using System.Collections.Generic;
+
+    public class SrcSrvContext
```

Comment:
I think we can put all options in the context:

```
public class SrcSrvContext
{
    public SrcSrvContext()
    {
        Paths = new List<Tuple<string, string>>();
        VstsData = new Dictionary<string, string>();
        
    }

    public string Revision { get; set; }

    public List<Tuple<string, string>> Paths { get; private set; }

    public Dictionary<string, string> VstsData { get; private set; }
}
```

Then we can add stuff in the future if we have to.